.github/workflows/test.yaml

```diff
@@ -99,18 +98,28 @@ jobs:
     - uses: twisted/python-info-action@v1
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip tox
+        python -m pip install --upgrade pip tox codecov coveralls
         tox --notest
 
     - name: Test
       run: |
         python --version
         tox -q
 
-    - name: Publish coverage
+    - name: Generate xMl and publish codecov.io
       # We want to publish coverage even on failure.
       if: contains(matrix['tox-env'], 'withcov') || failure()
-      run: tox -e coverage-prepare,codecov-push,coveralls-push
+      run: |
+        python -m coverage xml -o coverage.xml -i
+        python -m codecov -n 'lnx-${{ matrix.python-version }}-${{ matrix.tox-env }}${{ matrix.noipv6 }}' -X search -X gcov -f coverage.xml
```

Comment:
My understanding is that Codecov prefers use of the bash uploader.

Reply:
Thanks for the headup and the review.

I though that I have moved to bash, and this is why I haven't left another comment. My bad.

I will update this to use the bash uploader

from https://github.com/codecov/example-python

```
Q: What's the difference between the codecov-bash and codecov-python uploader?
A: As far as python is concerned, nothing. You may choose to use either uploader. Codecov recommends using the bash uploader when possible as it supports more unique repository setups. 
```